config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml

```diff
@@ -88,6 +88,10 @@ presubmits:
       repo: kubeadm
       base_ref: main
       path_alias: k8s.io/kubeadm
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: release-1.26
```

Comment:
1.26 because you want most recent stable? This will still have commits that aren't released yet vs a specific tag.
You'll have to make sure someone updates this periodically either way if not just using master.

Reply:
Good point. I've changed it to `master`.

Reply:
It would also be possible to do a version tag, I'm not sure what tradeoff you want here. master may force the repo to keep up with kubernetes changes aggressively but that can be better off for not falling behind, bit of a workflow / development style thing ...